tests/queries/0_stateless/02884_create_view_with_sql_security_option.sh

```diff
@@ -299,4 +301,16 @@ ${CLICKHOUSE_CLIENT} --user $user3 --query "INSERT INTO $db.session_events SELEC
 ${CLICKHOUSE_CLIENT} --user $user3 --query "SELECT count(*) FROM session_events"
 ${CLICKHOUSE_CLIENT} --query "SELECT count(*) FROM materialized_events"
 
+echo "===== TestOnCluster ====="
+${CLICKHOUSE_CLIENT} --multiquery <<EOF
+
+CREATE TABLE $db.test_cluster ON CLUSTER test_shard_localhost (a String) Engine = MergeTree() ORDER BY a FORMAT Null;
```

Comment:
Just for consistency: `Engine` is a keyword and we usually write keywords with capital letters (though they're parsed in case-insensitive manner)

Reply:
will fix it in a different PR